docs/Live-main/Track.py

```
@@ -0,0 +1,1445 @@
+        return ""
+
+    @property
+    def layout(self) -> RoutingChannelLayout:
+        """
+        Get the routing channel's Layout, e.g., mono or stereo.
+        """
+        return RoutingChannelLayout()
+
+
+class RoutingChannelVector(Vector):
+    def append(self, arg2: RoutingChannel) -> None:
```

Ruff (0.15.0)

[warning] Unused method argument: `arg2`

(ARG002)